src/_C014.ino

```diff
@@ -0,0 +1,686 @@
+              {
+                cmd = event->String2;
+                validTopic = true;
+              } else
+              {
+                cmd = "SYSTEM not found";
+              }
+            } else if (nodeName == F(CPLUGIN_014_CMD_VALUE)) // legethy homie/system/cmd topic
+            {
+              cmd = event->String2;
+              TempEvent.Source = VALUE_SOURCE_MQTT; // to trigger the correct feedback
+              validTopic = true;
+            } else // msg to a receiving plugin (?)
+            {
+              int deviceNr=CPlugin_014_getPluginNr(nodeName);
+              int pluginID=Device[getDeviceIndex(Settings.TaskDeviceNumber[deviceNr])].Number;
+
+              if (pluginID==12) // Plugin 12 LCD
+              {
+                cmd = valueName; // LCD or LCDCMD
+                cmd += F(",");
+                cmd += event->String2; // line,pos,Text or command as payload
+                validTopic = true;
+              } else if (pluginID==33) // Plugin 33 Dummy Device
+              { // DummyValueSet,<task/device nr>,<value nr>,<value/formula (!ToDo) >, works only with new version of P033!
+                int valueNr = CPlugin_014_getValueNr(deviceNr,valueName);
+                if (valueNr > -1) // value Name identified
+                {
+                  cmd = F("DummyValueSet,"); // Set a Dummy Device Value
+                  cmd += (deviceNr+1); // set the device Number
+                  cmd += F(",");
+                  cmd += (valueNr+1); // set the value Number
+                  cmd += F(",");
+                  cmd += event->String2; // expect float as payload!
+                  validTopic = true;
+                }
+              }
+            }
+
+            if (validTopic) {
+              parseCommandString(&TempEvent, cmd);
+              log+=F(" cmd: ");
+              log+=cmd;
+              addLog(LOG_LEVEL_INFO, log+ F(" OK"));
+            } else {
+              addLog(LOG_LEVEL_INFO, log+ F(" ERROR"));
+            }
+
+            // Feedback should not be sent here because here it will only confirm sucessfull retreval of the message!
+            // CPlugin_014_sendMQTTmsg(pubname, event->String2.c_str(), errorCounter);
+
+          }
+
+          if (validTopic) {
+            log=F("C014 : Exec > ");
+            // in case of event, store to buffer and return...
+            String command = parseString(cmd, 1);
+            if (command == F("event"))
+            {
+              log+=F("Event!");
+              eventBuffer = cmd.substring(6);
+            } else {
+              log=F("C014 : PluginCall:");
+              if (!PluginCall(PLUGIN_WRITE, &TempEvent, cmd)) {
+                remoteConfig(&TempEvent, cmd);
+                log +=F(" FALSE! remoteConfig?");
+              } else {
+                log +=F(" OK!");
+              }
+              addLog(LOG_LEVEL_INFO, log);
+            }
+
+          } else TempEvent.Source = 0; // something whent wrong
```

Comment:
What's being done with the `TempEvent` after this?
Is the else needed?
I would suspect something like `success = false;`, but in the rest of this case success is not handled at all.

Reply:
At first I tired to implement the acknowledgement via the TempEvent .Source as the JSON acknowledgement is send for TempEvent .Source=VALUE_SOURCE_MQTT. Think this was not the right way . Removed.